capstone/src/main/webapp/flags_script.js

```diff
@@ -209,12 +277,19 @@ function isPlaceClose(p1_lat, p1_lng, p2_lat, p2_lng){
     Math.sqrt(Math.sin(dlat / 2.0) * Math.sin(dlat / 2.0) + 
     Math.cos(p1_lat * rad)*Math.cos(p2_lat * rad) * 
     Math.sin(dlng / 2.0) * Math.sin(dlng / 2.0)));
-  console.log("Distance (mi): " + Math.floor(dist));
   if(Math.floor(dist) <= DISTANCE_THRESHOLD_MILES){
```

Comment:
Run a linter or better yet an automformatter on all files, not just the ones I commented on.